paper/paper.md

```diff
@@ -0,0 +1,108 @@
+  - microscopy
+  - imaging
+  - biology
+  - single cell biology
+  - spatial transcriptomics
+  - MERFISH
+  - In Situ Sequencing
+  - osmFISH
+  - smFISH
+  - BaristaSeq
+  - dartfish
+  - starmap
+  - seqFISH
+authors:
+  - name: Shannon Axelrod
+    affiliation: 1
+  - name: Matthew Cai
+    orcid: 0000-0003-4998-6328
+    affiliation: 1
+  - name: Ambrose J. Carr
+    orcid: 0000-0002-8457-2836
+    affiliation: 1
+  - name: Jeremy Freeman
+    orcid: 0000-0001-7077-7972
+    affiliation: 1
+  - name: Deep Ganguli
+    affiliation: 1
+  - name: Justin T. Kiggins
+    orcid: 0000-0002-4638-7015
+    affiliation: 1
+  - name: Brian Long
+    orcid: 0000-0002-7793-5969
+    affiliation: 2
+  - name: Tony Tung
+    affiliation: 1
+  - name: Kevin Yamauchi
+    orcid: 0000-0002-7818-1388
+    affiliation: 3
+affiliations:
+ - name: Chan Zuckerberg Initiative
+   index: 1
+ - name: Allen Institute
+   index: 2
+ - name: Chan Zuckerberg Biohub
+   index: 3
+date: 24 April 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+The exploding field of single cell transcriptomics has begun to enable deep analysis of gene expression and cell types, but spatial context is lost in the preparation of tissue for these assays.
+Recent developments in biochemistry, microfluidics, and microscopy have come together to bring about an “alphabet soup” of technologies that enable sampling gene expression in situ, with varying levels of spatial resolution, sensitivity, and genetic depth.
+These technologies promise to permit biologists to ask new questions about the spatial relationships between cell type and interactions between gene expression and cell morphology.
+However, these assays generate very large microscopy datasets which pose challenges for data processing using standard analysis tools in microscopy. Furthermore, many of these assays require specialized analysis to decode gene expression from multiplexed experimental designs.
+
+*starfish* is a Python library for processing images generated by microscopy-based spatial transcriptomics assays. It lets biologists build scalable pipelines that localize and quantify RNA transcripts in image data generated by any FISH or In Situ Sequencing method, from classic RNA single-molecule FISH to combinatorial barcoded assays. Image processing of an experiment is divided into fields of view (FOV) that correspond to the data produced by a microscope at a single location on a microscope slide. Built on top of scikit-image and xarray, *starfish* lets users register and pre-process images in each FOV, localize spots representing tagged RNA molecules in 3D, decode the identity of those molecules according to their experimental design, segment cells, assign the spots to cells, then aggregate spots into a cell x gene expression matrix. This spatially-annotated gene expression matrices can then be analyzed in downstream tools for single-cell biology, such as Seurat (@Seurat), Bioconductor (@Bioconductor), and ScanPy (@ScanPy).
```

Comment:
I think "In Situ Sequencing" should be "in situ sequencing" to refer to general methods that use sequencing rather than ISS. Or maybe "sequencing *in situ*" to really different differentiate from the specific ISS method.